packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts

```diff
@@ -0,0 +1,131 @@
+import {TestFixture} from '../../../fixtures/test-fixture';
+import {should} from '../../common-assertions';
+import {NumericFacetSelectors} from './numeric-facet-selectors';
+
+export function assertDisplayRangeInput(display: boolean) {
+  it(`${should(display)} display the range input`, () => {
+    NumericFacetSelectors.rangeInput().should(
+      display ? 'be.visible' : 'not.exist'
+    );
+  });
+}
+
+export function assertDisplayApplyButton(display: boolean) {
+  it(`${should(display)} display Apply button"`, () => {
+    NumericFacetSelectors.applyButton().should(
+      display ? 'be.visible' : 'not.exist'
+    );
+  });
+}
+
+export function assertDisplayInputWarning(length: number, message?: string) {
```

Comment:
Is it worthwhile to do those assertion on "input warning" `assertDisplayInputWarning`, `assertMinInputValue`, `assertMaxInputValue` ? 

Since they are handled by the browser, and I assume that the exact message will change for each browser ?

Reply:
we can check only if the 'warning message' appears then if it's too risky.
is it ok for you too @ThibodeauJF ?